hendrix/options.py

```diff
@@ -190,4 +196,5 @@ def options(argv=[]):
     A helper function that returns a dictionary of the default key-values pairs
     """
     parser = HendrixOptionParser
-    return vars(parser.parse_args(argv)[0])
+    parsed_arugs = parser.parse_args(argv)
+    return vars(parsed_arugs[0])
```

Comment:
Haha, `arugs`, and just the first one no less?

Reply:
Fixed in my subsequent PR (incoming).